src/components/member/tabs/contract-tab/contract-item.tsx

```diff
@@ -23,9 +33,21 @@ export const ContractItem: React.FunctionComponent<{
           <TerminationDate contract={contract} />
         </Card>
         <Card span={1}>
+          <span>
+            <strong>Market:</strong> {contract.market}
+          </span>
+        </Card>
+        {contract.agreements.map((agreement) => (
+          <AgreementItem agreement={agreement} contract={contract} />
+        ))}
+        <Card span={1}>
```

Comment:
I think `span={1}` is the default
```suggestion
        <Card>
```